pyvista/plotting/plotting.py

```diff
@@ -4725,12 +4725,10 @@ def add_point_scalar_labels(self, points, labels, fmt=None, preamble='', **kwarg
             raise TypeError(f'input points must be a pyvista dataset, not: {type(points)}')
         if not isinstance(labels, str):
             raise TypeError('labels must be a string name of the scalars array to use')
-        if fmt is None:
```

Comment:
Why was this removed?  It makes sense to me to use the theme format if `fmt` isn't explicitly given.  Is there something wrong with this logic?

Reply:
The reason is that the fmt is then overwritten with "%.6e". If fmt is None, the line is not working.
https://github.com/pyvista/pyvista/blob/c95debf226624195b16abac3b30a01d3676c0715/pyvista/plotting/plotting.py#L4728-L4731
Should we use `self._theme.font.fmt` instead of "%.6e"?

Reply:
I read those four lines like this:
1. if `fmt` is `None`, fall back to theme
2. if that is _also_ `None`, fall back to `'%.6e'`.

Reply:
I think it will only be overwritten if `self._theme.font.fmt` is also `None`.

Reply:
Ah! That makes sense. I revert it. Thanks!

Reply:
Resolved in 675e3cd43f0c6d3a4a6d3cb1f7f6bae5626174ea.